test/unit/clients/adminusers_client/user/get_user_test.js

```diff
@@ -34,44 +33,37 @@ describe('adminusers client - get user', () => {
   before(() => provider.setup())
   after(done => provider.finalize().then(done()))
 
-  selfServiceUserConfig.config.users.forEach(currentUser => {
-    describe(`success "${currentUser.cypressTestingCategory}" user`, () => {
-      const existingExternalId = currentUser.external_id
+  describe('success', () => {
```

Comment:
This describe could probably be more descriptive. I know it's a child of a larger block but something like 'find a valid user' might mean more to someone looking at this in isolation than 'success'

Reply:
good point 👍 , I've changed it